app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
+use App\Http\Requests\TaskRequest;
+use App\Task;
+use App\Http\Controllers\Controller;
+
+class TaskController extends Controller
+{
+    public function index()
+    {
+        $tasks = Task::orderBy('created_at', 'asc')->get();
+        
+        return view('tasks', [
+            'tasks' => $tasks
+        ]);
```

Comment:
Nghiên cứu https://laravel.com/docs/5.5/views#passing-data-to-views để xử lý chỗ này e nhé